modules/scratchFetch.ts

```diff
@@ -107,38 +101,32 @@ defineEvent("messageCreate", async (message) => {
 					embeds: [
 						{
 							title: data.username,
-							description: data.scratchteam ? "Scratch Team" : "",
-							color: 0x00ffff,
+							description: ``,
+							
 							fields: [
-								{
-									name: `WIWO`,
-									value: data.status,
-									inline: true,
-								},
 								{
 									name: `About`,
 									value: data.bio,
-									inline: true, //i tried to make this false but then it wouldnt send. instead of fixing the problem i ignored it
+									
 								},
+								{
+									name: `WIWO`,
+									value: data.status,
+									
+								},
+								
 							],
-							image: {
-								url: data.profile.images["90x90"],
-								height: 0,
-								width: 0,
-							},
+							
 							thumbnail: {
-								url: ``,
-								height: 0,
-								width: 0,
+								url: data.profile.images["90x90"],
+								
 							},
 							author: {
-								name: ``,
-								url: ``,
-								icon_url: ``,
+								name: data.scratchteam ? "Scratch Team" : "",
 							},
```

Comment:
```suggestion
...(data.scratchteam?{name:"Scratch Team"}: {}),
```
the way you have now will cause bugs i think

Reply:
this was 3 minutes ago how is it outdated

Reply:
idk just do it yourself ig